tests/snippets/testutils.py

```diff
@@ -1,36 +1,14 @@
-def assert_raises(exc_type, expr, msg=None):
-    """
-    Helper function to assert `expr` raises an exception of type `exc_type`.
-    Args:
-        expr: Callable
-        exec_type: Exception
-    Returns:
-        None
-    Raises:
-        Assertion error on failure
-    """
-    try:
-        expr()
-    except exc_type:
-        pass
-    else:
-        failmsg = '{} was not raised'.format(exc_type.__name__)
-        if msg is not None:
-            failmsg += ': {}'.format(msg)
-        assert False, failmsg
-
-
-def assertRaises(expected, *args, **kw):
+def assert_raises(expected, *args, **kw):
     if not args:
```

Comment:
I usually use the rule: "whenever an if statement contains a not, I swap the arms of the if statement".

In this case, you could write:
```python
if args:
   ...
else:
   ...
```

In my opinion, this is easier to understand for readers.

Reply:
Fixed it